lib/challenge.js

```diff
@@ -48,9 +63,17 @@ var pet_owner;
   // Every pet can:
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
-var some_pet;
+var some_pet {
 
 // Answer Starts Here
+Pet: cat,
+description: "Making an appointment for shots",
+name: Obie,
+species: cat,
+breed: siamese,
+noise: meow,
+var makeNoise = "Meow" ;
```

Comment:
This isn't how you write methods. Methods are just like any other property on an object, except that their values are functions.
```javascript
makeNoise : function(){
  console.log(this.noise);
}
```